### DISCUSSION TITLE
Writing a great plugin readme for WordPress.org

### DISCUSSION THREAD

**Discussion by ndiego** [WordPress/developer-blog-content#359] [Topic Ideas]
The official title is TBD, but the goal of this article is to provide a comprehensive guide on creating an effective readme file for plugins hosted in the WordPress.org [Plugins repository](https://wordpress.org/plugins/). Crafting a well-written readme is both an art and a skill, with many factors to consider. A definitive guide like this will serve as a valuable resource for both aspiring plugin developers and seasoned pros.

Props to @StevenDufresne for this topic idea.

**Comment by flexseth** [comment]
👋 I'd be interested in taking on this topic. 

> Fast track your plugin's approval to the WordPress.org plugin repository with a great README file.

Could be part of a larger resource on submitting to WP.org

**Comment by bph** [comment]
This topic was approved. Conversation continues on the issue
- #366
